code/modules/atmospherics/machinery/atmospherics.dm

```diff
@@ -297,7 +297,7 @@ Pipelines + Other Objects -> Pipe network
 			user.forceMove(target_move.loc) //handles entering and so on
 			user.visible_message("You hear something squeezing through the ducts.", "You climb out of the ventilation system.")
 		else if(target_move.can_crawl_through())
-			if(returnPipenet() != target_move.returnPipenet())
+			if(returnPipenet(target_move) != target_move.returnPipenet())
```

Comment:
The base def of returnPipenet returns null. Binary and Trinary devices return the pipenetwork of the pipes they are connected to when fed them as arguments. If we did what you sugested here it would break functionality in the case two binary devices are connected to eachother as they would both return the pipenetwork they share between them and thus they'd be equal. Theres no other point of feeding in an argument as all other returnPipenetwork procs dont require them.